src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -293,37 +271,41 @@ def test_start_job_status_loop__cell_ids(self):
 
                 self.jc.start_job_status_loop(init_jobs=True, cell_list=combo)
                 msg = self.jc._comm.last_message
-                self.assertEqual(
-                    {
-                        "msg_type": "job_status_all",
-                        "content": get_test_job_states(
-                            EXP_ALL_STATE_IDS
-                        ),  # consult version history for when this was exp_job_ids
-                    },
-                    msg["data"],
-                )
 
-                self.assertTrue(self.jc._running_lookup_loop)
-                self.assertTrue(self.jc._lookup_timer)
+                exp_job_ids = [
```

Comment:
Shuffled some things around here so it could fail fast and hopefully not keep the lookup thread running.

This also brings back some old code for getting the expected job ids - see the note on the left.

Reply:
thanks to @n1mus !